.changeset/blue-feet-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
```

Comment:
Think that this is a patch release right? as It's not changing the public API and it's not a breaking change? While we are below version 1.0.0 in semver minor releases are patches :)

Reply:
Ok perfect! I'll change this so. Thanks